src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
+table.tHead.rows[0].cells[3].after(th);
+table.tFoot.rows[0].cells[3].after(th.cloneNode(true));
+
+for (const tr of table.tBodies[0].rows) {
+  const td = document.createElement('td');
+
+  td.textContent = tr.cells[1].textContent;
+  tr.cells[3].after(td);
+}
```

Comment:
The loop assumes that every row in `tBodies[0]` has at least four cells. If any row has fewer cells, `tr.cells[3]` will be undefined, leading to an error. Consider adding a check to ensure that `tr.cells[3]` exists before attempting to insert after it.